### DISCUSSION TITLE
Compile once, mod anywhere?

### DISCUSSION THREAD

**Discussion by Hex4dec** [jart/cosmopolitan#1243] [Ideas]
Has anybody made something like the actual portable executable but for DLL injection/LD preload libraries? Could it even be feasible?

I'm not saying cosmopolitian should support this. I'm just seeing if anyone has similar projects so I wouldn't need to implement it myself.

Also, I'm not making this for any malicious script kiddie reasons for making cheats and hacks or anything, I'm making a game and I want to have a flexible engine modding system not limited by the bindings and hooks I give Lua. Kinda like how Minecraft's Mixin modding system works.

**Comment by jart** [comment]
It really isn't possible to do this, for a variety of reasons. I've looked into it. Can't be done sadly.
